sdk/core/Azure.Core/src/Serialization/ModelSerializerOptions.cs

```diff
@@ -9,16 +9,27 @@ namespace Azure.Core.Serialization
     /// <summary>
     /// Provides the client options for serializing models.
     /// </summary>
-    public class ModelSerializerOptions
+    public struct ModelSerializerOptions
     {
         /// <summary>
-        /// Initializes a new instance of the <see cref="ModelSerializerOptions" /> class. Takes in a string that determines Format of serialized model. "D" = data format which means IgnoreReadOnly and IgnoreAdditionalProperties are false, "W" = wire format which means both properties are true. Default is "D".
+        /// .
         /// </summary>
-        /// <param name="format"></param>
+        public static readonly ModelSerializerOptions AzureServiceDefault = new ModelSerializerOptions(ModelSerializerFormat.Wire);
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ModelSerializerOptions" /> class. Defaults to Data format "D".
+        /// </summary>
+        public ModelSerializerOptions() : this(ModelSerializerFormat.Data) { }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ModelSerializerOptions" /> class.
+        /// </summary>
+        /// <param name="format">String that determines Format of serialized model. "D" = data format which means IgnoreReadOnly and IgnoreAdditionalProperties are false, "W" = wire format which means both properties are true. Default is "D".</param>
         public ModelSerializerOptions(string format = "D")
```

Comment:
What is the intended approach for specifying JSON vs. JSON Merge Patch?  (vs. maybe even XML?)